ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -90,6 +90,62 @@ presubmits:
           - configMap:
               name: cluster-profile-aws
     trigger: ((?m)^/test( all| e2e-aws),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-smoke
+    decorate: true
+    name: pull-ci-origin-installer-e2e-aws-smoke
+    rerun_command: /test e2e-aws-smoke
+    # The abomination below is equivalent to `^((?!Documentation).)*$`. Since
+    # Go doesn't support negative lookaheads, we are stuck with the following.
+    run_if_changed: ^([^D]|D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*([^Do]|o[^Dc]|oc[^Du]|ocu[^Dm]|ocum([^De]|e([^Dn]|n([^Dt]|t([^Da]|a[^Dt]|at[^Di]|ati[^Do]|atio[^Dn]))))))*(D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*(o|oc|ocu|ocum(e(n(t(a|at|ati|atio)?)?)?)?)?)?$
+    skip_cloning: true
+    spec:
+      containers:
+      - command:
+        - ci-operator
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-aws-smoke-cluster-profile
+        - --template=/usr/local/e2e-aws-smoke
+        - --target=e2e-aws-smoke
+        env:
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-smoke
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.json
```

Comment:
This needs to be changed to `master.yaml`.

We have mass-renamed all ci-operator configuration files from `.json` to `.yaml` in https://github.com/openshift/release/pull/1336, because YAML is what ci-operator consumes now. All the references of these files needed to be changed too. JSON is a YAML subset so that the actual file content can stay like it is (we'll eventually mass-reformat them to the block style).

Reply:
rebased, thanks

Reply:
and rebased against block style, too lol